quick-start-with-tidb.md

```diff
@@ -6,510 +6,532 @@ aliases: ['/docs/dev/quick-start-with-tidb/','/docs/dev/test-deployment-using-do
 
-        > **Note:**
-        >
-        > For the playground operated in this way, after the test deployment is finished, TiUP will clean up the original cluster data. You will get a new cluster after re-running the command.
-        > If you want the data to be persisted on storage, run `tiup --tag <your-tag> playground ...`. For details, refer to the [TiUP Reference](/tiup/tiup-reference.md#-t---tag) guide.
+```shell
+tiup playground v8.1.0 --tag sample-app-env --db 2 --kv 3 --pd 1 --tiflash 0
+```
 
-4. Start a new session to access TiDB:
+TiUP will download the necessary binaries for the specified version and start the cluster components for you.
 
-    + Use the TiUP client to connect to TiDB.
+### Manage your playground environments
 
-        {{< copyable "shell-regular" >}}
+You can list all your playground environments with:
 
-        ```shell
-        tiup client
-        ```
+```shell
+tiup status
+```
 
-    + Alternatively, you can use the MySQL client to connect to TiDB.
+You will see an output similar to this:
 
-        {{< copyable "shell-regular" >}}
+```log
+Name            Component  PID    Status  ... Args
+sample-app-env  playground 30661  RUNNING ... tiup-playground v8.1.0 --tag sample-app-env --db 2 --kv 3 --pd 1 --tiflash 0
+```
 
-        ```shell
-        mysql --host 127.0.0.1 --port 4000 -u root
-        ```
+If you stop a tagged playground cluster by pressing <kbd>Ctrl+C</kbd> in the terminal where you started it, you can start it again without losing data.
 
-5. Access the Prometheus dashboard of TiDB at <http://127.0.0.1:9090>.
+To restart the environment, you can run `tiup playground` with the same arguments as in the `tiup status` output, as seen in the output above, or you can change them. This means that you have the flexibility to add and remove cluster components.
 
-6. Access the [TiDB Dashboard](/dashboard/dashboard-intro.md) at <http://127.0.0.1:2379/dashboard>. The default username is `root`, and the password is empty.
+To add a TiProxy instance as a load balancer to your existing `sample-app-env` environment with:
 
-7. Access the Grafana dashboard of TiDB through <http://127.0.0.1:3000>. Both the default username and password are `admin`.
 
-8. (Optional) [Load data to TiFlash](/tiflash/tiflash-overview.md#use-tiflash) for analysis.
+```shell
+tiup playground v8.1.0 --tag sample-app-env --db 2 --kv 3 --pd 1 --tiflash 0 --tiproxy 1
+```
 
-9. Clean up the cluster after the test deployment:
+You should see the cluster starting normally, with an output like this:
 
-    1. Stop the process by pressing <kbd>Control+C</kbd>.
+```log
+TiDB Playground Cluster is started, enjoy!
+Connect TiDB: mysql --host 127.0.0.1 --port 4001 -u root
+Connect TiDB: mysql --host 127.0.0.1 --port 4000 -u root
+Connect TiProxy: mysql --host 127.0.0.1 --port 6000 -u root
```

Comment:
```suggestion
Connect TiProxy: mysql --host 127.0.0.1 --port 6000 -u root
TiDB Dashboard:  http://127.0.0.1:60434/dashboard
Grafana:         http://127.0.0.1:60838
```

Reply:
reason: We can instruct users to note the Dashboard link as it will be used in "TiDB Dashboard: SQL insights" to check "weather data queries"